### PR TITLE
Update redirected URLs happening due to subdomain change (part 25)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updated redirected URLs.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/server-side/django/deployment/index.md [hunk 1]
```diff
@@ -561,9 +561,9 @@ You can check out the add-ons to your app using the `heroku addons` command. Thi
 ```bash
 > heroku addons
 
-Add-on                                     Plan       Price  State
+Add-on Plan Price State
```
Comment (teoli2003) [inline files/en-us/learn/server-side/django/deployment/index.md:564]: I don't think we should change this. It looks like the output of the command and is outside the scope of the PR.

File: files/en-us/learn/server-side/apache_configuration_htaccess/index.md [hunk 2]
```diff
@@ -165,60 +165,60 @@ Servers should use text/javascript for JavaScript resources as indicated in the
 ```bash
 <IfModule mod_expires.c>
   # Data interchange
-    AddType application/atom+xml      atom
```
Comment (teoli2003) [inline files/en-us/learn/server-side/apache_configuration_htaccess/index.md:168]: I don't want to change this in this PR. If you think it is correct to change this, please file a separate PR. Thanks.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
@teoli2003 done!
That was an accidental change.

File: files/en-us/learn/server-side/apache_configuration_htaccess/index.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ The possible values for the first parameter are listed below. If the first param
 
 ## Cross-origin resources
 
-The first set of directives control [CORS](https://fetch.spec.whatwg.org/) (Cross-Origin Resource Sharing) access to resources from the server. CORS is an HTTP-header based mechanism that allows a server to indicate the external origins (domain, protocol, or port) which a browser should permit loading of resources.
+The first set of directives control [CORS]((https://fetch.spec.whatwg.org/) (Cross-Origin Resource Sharing) access to resources from the server. CORS is an HTTP-header based mechanism that allows a server to indicate the external origins (domain, protocol, or port) which a browser should permit loading of resources.
```
Comment (teoli2003) [inline files/en-us/learn/server-side/apache_configuration_htaccess/index.md:59]: I think this `(` has to go.

File: files/en-us/learn/server-side/django/deployment/index.md [hunk 4]
```diff
@@ -588,7 +588,7 @@ DATABASE_URL: postgres://uzfnbcyxidzgrl:j2jkUFDF6OGGqxkgg7Hk3ilbZI@ec2-54-243-20
 
 If you recall from the section on [getting the website ready to publish](#getting_your_website_ready_to_publish), we have to set environment variables for `DJANGO_SECRET_KEY` and `DJANGO_DEBUG`. Let's do this now.
 
-> **Note:** The secret key needs to be really secret! One way to generate a new key is to use the [Django Secret Key Generator](https://miniwebtool.com/django-secret-key-generator/).
+> **Note:** The secret key needs to be really secret! One way to generate a new key is to use the [Django Secret Key Generator](https://www.miniwebtool.com/django-secret-key-generator/).
```
Comment (teoli2003) [inline files/en-us/learn/server-side/django/deployment/index.md:591]: This change was ok

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two last bits and we can merge this.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**